src/config.py

```diff
@@ -34,8 +54,10 @@
     environment = 'KDE Plasma'
 elif os.getenv('XDG_CURRENT_DESKTOP') == 'Deepin':
     environment = 'Deepin'
-else:
-    from tty_environments import *
+elif os.getenv('XDG_CURRENT_DESKTOP') == 'Hyprland':
+    environment = 'Hyprland'
+#else:
+#    from tty_environments import *
```

Comment:
The tty_environments.py file is used to detect DE if SaveDesktop is running in TTY mode (i.e. pure CLI without graphical environment or tile WM)

Reply:
i do not know why i deleted that ......

Reply:
done